CMakeLists.txt

```diff
@@ -249,6 +249,7 @@ if(HIP_PLATFORM STREQUAL "hcc")
         src/hip_texture.cpp
         src/hip_surface.cpp
         src/hip_intercept.cpp
+        src/hip_launch_kernel.cpp
```

Comment:
Do we really need a new file?

Reply:
I could not findout any relevant place to add hence created new file. Let me know if you have nay suggestion.